dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -503,9 +512,16 @@ export default defineComponent({
         path: `/companies/${thisCompanyId}`,
       });
     },
+    /**
+     * Updates the hasCompanyDataOwner in an async way
+     */
+    async updateHasCompanyDataOwner() {
+      this.hasCompanyDataOwner = await hasCompanyAtLeastOneDataOwner(this.companyIdentifier, this.getKeycloakPromise);
+    },
   },
   mounted() {
     this.retrieveFrameworkOptions();
+    void this.updateHasCompanyDataOwner();
```

Comment:
sonar issue: try this.updateHasCompanyDataOwner().catch((error) => console.error(error));

Reply:
probably the void is the problem. I first have to verify that the functionality in the single data request form actually works.